edb/server/cluster.py

```diff
@@ -155,7 +155,7 @@ def connect(self, **kwargs):
 
         return edgedb.connect(**connect_args)
 
-    def init(self, *, server_settings={}):
+    def init(self, *, server_settings=None):
```

Comment:
Since `server_settings` isn't used in the body of this method at all, it can very well be None with no supporting code to transform this into a dictionary.